src/components/VideoRating.tsx

```diff
@@ -6,21 +6,35 @@ import { useState } from "react";
 interface VideoRatingProps {
   ratingProp?: number;
   votesCount?: number;
+  onGetNewRating: (rating: number) => void;
+  isAuthorized: boolean;
+  isRatingFreezen: boolean;
 }
-const VideoRating = ({ ratingProp, votesCount }: VideoRatingProps) => {
+const VideoRating = ({
+  ratingProp,
+  votesCount,
+  onGetNewRating,
+  isAuthorized,
+  isRatingFreezen,
+}: VideoRatingProps) => {
```

Comment:
Please, create component RatingStar and use it in VideoRating component to render star